engines/dradis-api/config/routes.rb

```diff
@@ -1,8 +1,17 @@
 Dradis::CE::API::Engine::routes.draw do
+  concern :with_comments do
+    resources :comments, only: [:index, :create]
+  end
+
   scope path: :api do
     defaults format: 'json' do
       scope module: :v1, constraints: Dradis::CE::API::RoutingConstraints.new(version: 1, default: true) do
-        resources :issues
+        resources :issues, concerns: :with_comments
+        resources :evidence, only: [], concerns: :with_comments
```

Comment:
Another resource line is added for notes and evidence to prevent an unnecessarily long path. Ex:
```
/api/nodes/1/notes/1/comments
```
vs
```
/api/notes/1/comments
```

Reply:
I am good with the long path version, I feel its is more consistent.
If I see `/api/notes/1/comments`, makes me think that I can access the note with: `/api/notes/1`, but that is not true.

Reply:
@dormi that would mean that when deleting comments, you'd need 3 id's (node, commentable, comment). Is that okay? Or should we have an `/api/comments` path?